packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -455,7 +456,17 @@ const ProductDetail = () => {
         >
             <Helmet>
                 <title>{product?.pageTitle}</title>
-                <meta name="description" content={product?.pageDescription} />
+                {product?.pageMetaTags ? (
+                    // description is one of the page meta tags returned for products
+                    // and this is the same as product?.pageDescription
+                    product.pageMetaTags?.map((pageMetaTag) => {
+                        const name = pageMetaTag.id
+                        const content = pageMetaTag.value
+                        return <meta name={name} content={content} key={name} />
+                    })
+                ) : (
+                    <meta name="description" content={product?.pageDescription} />
```

Comment:
Similarly, I think we should also consider the scenario where the `pageMetaTags` does not include a `description` key. In that case, the code will fall back to using the pageDescription.

```suggestion
                {product?.pageMetaTags?.length > 0 && product.pageMetaTags.map(({ id, value }) => (
                    <meta name={id} content={value} key={id} />
                ))}
                {/* Fallback for description if not included in pageMetaTags */}
                {!product?.pageMetaTags?.some(tag => tag.id === 'description') && product?.pageDescription && (
                    <meta name="description" content={product.pageDescription} />
```

Reply:
My understanding is that the description meta tag is generated from the product description so they will always be the same value. But I can add this fallback just in case.